app/scripts/background.js

```diff
@@ -273,6 +273,15 @@ function saveTimestamp() {
  * @returns {Promise} Setup complete.
  */
 async function initialize() {
+  try {
+    browser.privacy.network.networkPredictionEnabled.set({
```

Comment:
maybe this should specifically target `chrome` rather than the `browser` polyfill